pkgs/development/tools/rust/cargo-deadlinks/default.nix

```
@@ -16,6 +16,9 @@ rustPlatform.buildRustPackage rec {
   checkFlags = [
     # uses internet
     "--skip non_existent_http_link --skip working_http_check"
+    # makes assumption about HTML paths that changed in rust 1.82.0
+    "--skip simple_project::it_checks_okay_project_correctly"
+    "--skip cli_args::it_passes_arguments_through_to_cargo"
```

(Better to, but not required here)

```suggestion
    "--skip=cli_args::it_passes_arguments_through_to_cargo"
```